cmd/root.go

```diff
@@ -101,8 +104,19 @@ along with the synchronized files.`,
 
 		c := kubectl.NewClient(config)
 
+		nodeSelectors := make(map[string]string)
+		if len(opt.NodeSelectorKeys) > 0 {
+			if len(opt.NodeSelectorKeys) != len(opt.NodeSelectorValues) {
+				fmt.Fprintln(stderr, "Node selector keys and values don't match, dropping node selector. %d != %d", len(opt.NodeSelectorKeys), len(opt.NodeSelectorValues))
```

Comment:
I believe that in case of invalid parameters, the whole command should fail.
So return error here.

Reply:
With the change you suggested below, this block of code has been removed.